### ISSUE TITLE
watchdog: TypeError: an integer is required

### ISSUE THREAD

**Issue by DeathBorn** [zalando/patroni#530]
Centos 7, patroni 1.3.3, python2
```
Sep 23 07:10:16 pgdron001d1vteo patroni[2796]: 2017-09-23 07:10:16,595 INFO: Zookeeper connection established, state: CONNECTED
Sep 23 07:10:16 pgdron001d1vteo patroni[2796]: Traceback (most recent call last):
Sep 23 07:10:16 pgdron001d1vteo patroni[2796]: File "/usr/bin/patroni", line 11, in <module>
Sep 23 07:10:16 pgdron001d1vteo patroni[2796]: load_entry_point('patroni==1.3.3', 'console_scripts', 'patroni')()
Sep 23 07:10:16 pgdron001d1vteo patroni[2796]: File "/usr/lib/python2.7/site-packages/patroni/__init__.py", line 173, in main
Sep 23 07:10:16 pgdron001d1vteo patroni[2796]: return patroni_main()
Sep 23 07:10:16 pgdron001d1vteo patroni[2796]: File "/usr/lib/python2.7/site-packages/patroni/__init__.py", line 142, in patroni_main
Sep 23 07:10:16 pgdron001d1vteo patroni[2796]: patroni.run()
Sep 23 07:10:16 pgdron001d1vteo patroni[2796]: File "/usr/lib/python2.7/site-packages/patroni/__init__.py", line 114, in run
Sep 23 07:10:16 pgdron001d1vteo patroni[2796]: logger.info(self.ha.run_cycle())
Sep 23 07:10:16 pgdron001d1vteo patroni[2796]: File "/usr/lib/python2.7/site-packages/patroni/ha.py", line 1056, in run_cycle
Sep 23 07:10:16 pgdron001d1vteo patroni[2796]: info = self._run_cycle()
Sep 23 07:10:16 pgdron001d1vteo patroni[2796]: File "/usr/lib/python2.7/site-packages/patroni/ha.py", line 1029, in _run_cycle
Sep 23 07:10:16 pgdron001d1vteo patroni[2796]: msg = self.process_healthy_cluster()
Sep 23 07:10:16 pgdron001d1vteo patroni[2796]: File "/usr/lib/python2.7/site-packages/patroni/ha.py", line 684, in process_healthy_cluster
Sep 23 07:10:16 pgdron001d1vteo patroni[2796]: if self.update_lock(True):
Sep 23 07:10:16 pgdron001d1vteo patroni[2796]: File "/usr/lib/python2.7/site-packages/patroni/ha.py", line 90, in update_lock
Sep 23 07:10:16 pgdron001d1vteo patroni[2796]: self.watchdog.keepalive()
Sep 23 07:10:16 pgdron001d1vteo patroni[2796]: File "/usr/lib/python2.7/site-packages/patroni/watchdog/base.py", line 36, in wrapped
Sep 23 07:10:16 pgdron001d1vteo patroni[2796]: return func(self, *args, **kwargs)
Sep 23 07:10:16 pgdron001d1vteo patroni[2796]: File "/usr/lib/python2.7/site-packages/patroni/watchdog/base.py", line 205, in keepalive
Sep 23 07:10:16 pgdron001d1vteo patroni[2796]: self.impl.keepalive()
Sep 23 07:10:16 pgdron001d1vteo patroni[2796]: File "/usr/lib/python2.7/site-packages/patroni/watchdog/linux.py", line 190, in keepalive
Sep 23 07:10:16 pgdron001d1vteo patroni[2796]: os.write(self._fd, b'1')
Sep 23 07:10:16 pgdron001d1vteo patroni[2796]: TypeError: an integer is required
```

**Comment by CyberDem0n** [comment]
This one was definitely fixed in 1.3.4

**Comment by DeathBorn** [comment]
Oh! My bad then.
